bundles/core/src/test/resources/title/exporter-title-jcr-title-link-v2.json

```diff
@@ -1,13 +1,16 @@
 {
     "id": "title-6032c7e80a",
     "linkDisabled": false,
-    "linkURL": "https://www.adobe.com",
     "dataLayer": {
         "title-6032c7e80a": {
             "repo:modifyDate": "2016-01-13T15:14:51Z",
             "xdm:linkURL": "https://www.adobe.com",
             "@type": "core/wcm/components/title/v2/title"
         }
     },
+    "link": {
```

Comment:
As `getLink` should not be introduced in v1 (see comment above), the `link` property won't be part of the JSON rendering in v1. This applies here and below.

Reply:
also, this json is for v2